pages/_app.tsx

```diff
@@ -95,12 +99,19 @@ const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
                         <GraphProvider>
                             <FactoryStore>
                                 <TransactionStore>
-                                    <Component {...pageProps} />
+                                    <SelectedTracerStore>
+                                        <OMEStore>
+                                            <InsuranceStore>
+                                                <OrderStore>
+                                                    <Component {...pageProps} />
+                                                </OrderStore>
+                                            </InsuranceStore>
+                                        </OMEStore>
+                                    </SelectedTracerStore>
```

Comment:
This will load the InsuranceStore twice on the InsurancePage. Am generally against this as we would be using the entire Context Store on all pages when really you only need a couple of things from each of them. that could be implemented in hooks. If you let me know what you need I will think up what the best way to do it would be